linkis-public-enhancements/linkis-bml/linkis-bml-client/src/main/scala/org/apache/linkis/bml/common/POSTActionFailException.scala

```diff
@@ -20,12 +20,13 @@ package org.apache.linkis.bml.common
 import org.apache.linkis.common.exception.ErrorException
 
 case class POSTActionFailException()
-    extends ErrorException(70025, "material house client request failed (物料库客户端请求失败)") {}
+    extends ErrorException(70025, "material house client request failed(物料库客户端请求失败)") {}
 
 case class POSTResultNotMatchException()
     extends ErrorException(
       70021,
-      "The result returned by the repository client POST request does not match(物料库客户端POST请求返回的result不匹配)"
+      "The result returned by the repository client POST request does not match" +
```

Comment:
File line length exceeds 100 characters  
Is it possible to adjust the default value 100